projects/observability/src/shared/dashboard/widgets/donut/donut-widget.model.ts

```diff
@@ -65,13 +69,25 @@ export class DonutWidgetModel {
   })
   public header?: WidgetHeaderModel;
 
+  @ModelProperty({
+    key: 'color-palette',
+    type: ARRAY_PROPERTY.type,
+    required: false
+  })
+  public colorPalette: string[] = [];
+
   @ModelInject(MODEL_API)
   private readonly api!: ModelApi;
 
+  @ModelInject(ColorService)
+  private readonly colorService!: ColorService;
+
   public getData(): Observable<DonutResults> {
+    this.colorService.registerColorPalette(DonutWidgetModel.DONUT_COLORS, this.colorPalette);
```

Comment:
Don't need to register this and it doesn't really make sense since it's local to this instance of this mode.